packages/gestalt/src/Popover/Contents.js

```diff
@@ -86,60 +88,66 @@ export default function Contents({
+  const contents = (
+    <div
+      ref={refs.setFloating}
+      tabIndex={-1}
+      className={classnames(
+        styles.container,
+        rounding === 2 && borders.rounding2,
+        rounding === 4 && borders.rounding4,
+        styles.contents,
+        styles.maxDimensions,
+        width !== null && styles.minDimensions,
+      )}
+      style={{ ...floatingStyles, visibility }}
+    >
+      {caret && (
+        <div
+          ref={caretRef}
+          className={classnames(colors[bgColorElevated], styles.caret)}
+          style={{
+            left: caretOffset?.x != null ? `${caretOffset.x}px` : '',
+            top: caretOffset?.y != null ? `${caretOffset.y}px` : '',
+            [placement]: '100%',
+          }}
+        >
+          <Caret
+            direction={SIDES_MAP[placement]}
+            height={isCaretVertical ? CARET_HEIGHT : CARET_WIDTH}
+            width={isCaretVertical ? CARET_WIDTH : CARET_HEIGHT}
+          />
+        </div>
+      )}
+
       <div
-        ref={refs.setFloating}
-        tabIndex={-1}
+        aria-label={accessibilityLabel}
+        id={id}
+        role={role}
         className={classnames(
-          styles.container,
+          border && styles.border,
+          colors[background],
+          colors[bgColorElevated],
           rounding === 2 && borders.rounding2,
           rounding === 4 && borders.rounding4,
-          styles.contents,
+          styles.innerContents,
           styles.maxDimensions,
           width !== null && styles.minDimensions,
         )}
-        style={{ ...floatingStyles, visibility }}
+        style={{
+          maxWidth: width,
+        }}
       >
-        {caret && (
-          <div
-            ref={caretRef}
-            className={classnames(colors[bgColorElevated], styles.caret)}
-            style={{
-              left: caretOffset?.x != null ? `${caretOffset.x}px` : '',
-              top: caretOffset?.y != null ? `${caretOffset.y}px` : '',
-              [placement]: '100%',
-            }}
-          >
-            <Caret
-              direction={SIDES_MAP[placement]}
-              height={isCaretVertical ? CARET_HEIGHT : CARET_WIDTH}
-              width={isCaretVertical ? CARET_WIDTH : CARET_HEIGHT}
-            />
-          </div>
-        )}
-
-        <div
-          aria-label={accessibilityLabel}
-          id={id}
-          role={role}
-          className={classnames(
-            border && styles.border,
-            colors[background],
-            colors[bgColorElevated],
-            rounding === 2 && borders.rounding2,
-            rounding === 4 && borders.rounding4,
-            styles.innerContents,
-            styles.maxDimensions,
-            width !== null && styles.minDimensions,
-          )}
-          style={{
-            maxWidth: width,
-          }}
-        >
-          {children}
-        </div>
+        {children}
       </div>
+    </div>
+  );
+
+  return role === 'tooltip' ? (
```

Comment:
Could we just set when this is a tooltip = -1
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/2e559534-2d31-4074-9142-1af2de19a602)

Reply:
FloatingFocusManager adds inert attribute to all the other elements which is not desirable. That's unconditional. That's why removing FloatingFocusManager is necessary for tooltip. Besides it doesn't make sense to have focus manager for a component that should not have focus.

Reply:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/db1407ae-017d-41f3-87bd-761d78931162)

Reply:
https://floating-ui.com/docs/floatingfocusmanager#disabled